sdk/storage/inc/datalake/datalake_options.hpp

```diff
@@ -18,23 +20,42 @@ namespace Azure { namespace Storage { namespace DataLake {
    */
   struct ServiceClientOptions
   {
-    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> policies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerOperationPolicies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerRetryPolicies;
   };
 
   /**
    * @brief File system client options used to initalize DataLakeFileSystemClient.
    */
   struct FileSystemClientOptions
   {
-    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> policies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerOperationPolicies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerRetryPolicies;
   };
 
   /**
    * @brief Path client options used to initalize DataLakePathClient.
    */
   struct PathClientOptions
   {
-    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> policies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerOperationPolicies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerRetryPolicies;
+  };
+
+  /**
+   * @brief Specifies access conditions for a file system.
+   */
+  struct FileSystemAccessConditions : public LastModifiedTimeAccessConditions
```

Comment:
Are you sure all filesystem operations support these 2 access conditions?

Reply:
Yes, all filesystem operation that supports access condition supports exactly those two access conditions.